### DISCUSSION TITLE
Novelas TV

### DISCUSSION THREAD

**Discussion by Virginair-corp** [iptv-org/community#649] [Q&A]
**Channel Name:** novelas tv
**Channel ID:** NovelasTV.fr
**Country:** fr
**Language:** fr
**External Link (optional):** xxx
**Notes (optional):** xxx

**Comment by yassine426** [comment]
Bein sports iptv arabic


Le mer. 10 mars 2021 à 00:29, Virginair-corp <notifications@github.com> a
écrit :

**Comment by Sphinxroot** [comment]
#EXTINF:-1 tvg-id="5f914f9dccb4de0007df8bc4" tvg-chno="340" tvg-logo="https://images.pluto.tv/channels/5f914f9dccb4de0007df8bc4/colorLogoPNG.png" group-title="Séries",Pluto TV Telenovela
https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5f914f9dccb4de0007df8bc4/master.m3u8?deviceId=channel&deviceModel=web&deviceVersion=1.0&appVersion=1.0&deviceType=rokuChannel&deviceMake=rokuChannel&deviceDNT=1&advertisingId=channel&embedPartner=rokuChannel&appName=rokuchannel&is_lat=1&bmodel=bm1&content=channel&platform=web&tags=ROKU_CONTENT_TAGS&coppa=false&content_type=livefeed&rdid=channel&content_rating=ROKU_ADS_CONTENT_RATING

si t en veut d autre jai  toute les nouveautes dans mon git section france

**Comment by Virginair-corp** [comment]
Bonjour et merci bien , j'ai essayé de regarder mais ce n'ai pas novelas
TV. Essai possible d'avoir surtout de l'Afrique ?

Le mer. 10 mars 2021 à 21:11, Sphinxroot ***@***.***> a
écrit :

**Comment by BellezaEmporium** [comment]
> 
> 
> **Channel Name:** novelas tv
> **Country:** fr
> **Language:** fr
> **External Link (optional):** xxx
> **Notes (optional):** xxx

@Sphinxroot Cette chaîne est un peu partout chez les fournisseurs d'accès. Avec un peu de chance, tu la trouvera chez Orange. Eux et leurs streams "pas vraiment" protégés...

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19651
